docs/networking/best-practice-non-vlan-aware-switch.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 51
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Network
+  - network
+  - VLAN
+  - vlan
+Description: Harvester is built on top of Kubernetes, and uses the [CNI](https://github.com/containernetworking/cni) as the interface between network providers and Kubernetes pod networking. Naturally, we implement the Harvester network based on CNI. Moreover, the Harvester UI integrates the network configuration in order to provide a user-friendly way to configure networks for VMs.
+---
+
+# Best Practice for Non VLAN-aware Switch
+
+In this best practice guide for "non VLAN-aware" switch, also known as "dummy" switch, we will introduce Harvester VLAN network and external switch configuration for common scenario.
+
+## Architecture
+
+Hardware:
+- Three Harvester servers with only one single port network card.
+- One or more "non VLAN-aware" switch(es).
+
+Network Specification:
+- The host and the VM networks are in the same subnet.
+
+Cabling:
+- The Harvester servers are connected to the switch in a port from `1` to `3`.
+
+The following diagram illustrates the cabling used for this guide:
+
+   ![non-vlan-aware-case.png](assets/non-vlan-aware-case.png)
+
+## External Switch Configuration
+
+Typically, a "non VLAN-aware" switch cannot be configured.
+
+## Create a VLAN Network in Harvester
+
+You can create a new VLAN network in the **Advanced > Networks** page, and click the **Create** button.
+
+Specify the name and VLAN ID that you want to create for the VLAN network <small>(You can specify the same VLAN ID in different namespaces if you have [Rancher multi-tenancy](/rancher/virtualization-management/#multi-tenancy) configured)</small>.
+   ![create-vlan-network.png](assets/create-network.png)
+
+### Connect a VM to the Harvester management network
```

Comment:
What does `the Harvester management network` mean here?  Usually, in Harvester, we take the network provided by the canal as the management network.

Reply:
the Harvester management network is equal to the subnet of `harvester-mgmt`

Reply:
No. They are not the same in the Harvester. Please refer https://docs.harvesterhci.io/v1.0/networking/harvester-network/#management-network.

Reply:
i mean
the term "mgmt network" in this docs is same as the subnet of harvester-mgmt

maybe i need to change the term to different words
@nunix do you have any suggestion for this term?

Reply:
We have used `management network` to indicate the network provided by the canal. It would be confusing to use the same phrase with a different meaning. A simple suggestion is just calling it `the subnet of the Harvester hosts` or `the Harvester local network`.

Reply:
ok fix it with `the subnet of the Harvester hosts`